test/alternator/test_table.py

```diff
@@ -553,3 +554,55 @@ def creates():
                     time.sleep(1)
                     continue
                 raise
+
+# Test that DeleteTable returns correct TableDescription (issue #11472)
+def test_delete_table_description(dynamodb):
+    table_name = 'alternator_test'
+
+    table = create_table(dynamodb, table_name)
```

Comment:
After you call create_table() and before you can delete it, you need to wait for the table to have been created (this is not important on Alternator, but it's important on DynamoDB - where this CREATING phase can last many seconds). See an example in the create_and_delete_table() function.

You can use the create_test_table() utility function, which picks a unique table name (using unique_table_name() instead of 'alternator_test') and waits for the creation to complete.

Reply:
Thanks, I used test_table.py:create_and_delete_table function as example. Done.